src/DataMapper/OrdersUserDataDataMapper.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusFacebookPlugin\DataMapper;
+
+use FacebookAds\Object\ServerSide\Gender;
+use function Safe\preg_replace;
+use Setono\SyliusFacebookPlugin\ServerSide\ServerSideEventInterface;
+use Sylius\Component\Core\Model\AddressInterface;
+use Sylius\Component\Core\Model\Customer;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Webmozart\Assert\Assert;
+
+/* not final */ class OrdersUserDataDataMapper implements DataMapperInterface
```

Comment:
`/* not final */` is this for easy extension?

Reply:
Yeah, exactly. So plugin users can just extend data mapper and do what they need very fast.